rsqueakvm/model/compiled_methods.py

```diff
@@ -184,8 +191,15 @@ def getliteral(self, index):
     def primitive(self):
         return self._primitive
 
+    def frame_size(self):
+        return self._frame_size
+
+    def update_frame_size(self, size):
```

Comment:
Frame sizes will only grow. This isn't perfect, because some loops might not require the whole frame, but this way we'll never end up with some assembler that tries to read more frame than what's there. I'm not certain this might be too conservative, though